pom.xml

```diff
@@ -145,13 +179,114 @@
+                                        <exclude>org.slf4j:slf4j-nop</exclude>
+                                        <exclude>org.slf4j:slf4j-jdk14</exclude>
+                                        <exclude>org.slf4j:slf4j-log4j12</exclude>
+                                        <exclude>org.slf4j:slf4j-log4j13</exclude>
+                                        <!-- Ban Logback implementations -->
+                                        <exclude>ch.qos.logback:logback-classic</exclude>
+                                        <exclude>ch.qos.logback:logback-core</exclude>
+                                        <exclude>ch.qos.logback:logback-access</exclude>
+                                        <exclude>ch.qos.logback:logback</exclude>
+                                        <exclude>org.osgi:org.osgi.annotation.versioning</exclude>
+                                        <!-- Ban Spring Dependencies (since we have our own jars)-->
+                                        <exclude>org.springframework:spring-core</exclude>
+                                        <exclude>org.springframework:spring-beans</exclude>
+                                        <exclude>org.springframework:spring-context</exclude>
+                                        <exclude>org.springframework:spring-web</exclude>
+                                        <exclude>org.springframework:spring-webmvc</exclude>
+                                        <exclude>org.springframework.data:spring-data-jpa</exclude>
+                                        <exclude>org.springframework.data:spring-data-commons</exclude>
+                                        <exclude>org.springframework.security:spring-security-core</exclude>
+                                        <exclude>org.springframework.boot:spring-boot</exclude>
+                                        <!-- Ban checker-qual, we don't use Checker Framework -->
+                                        <exclude>org.checkerframework:checker-qual</exclude>
+                                        <!-- We use our own impl here, including this one causes problems-->
+                                        <exclude>org.jboss.resteasy:resteasy-context-propagation</exclude>
+                                        <exclude>com.google.android:annotations</exclude>
+                                        <exclude>org.codehaus.mojo:animal-sniffer-annotations</exclude>
+                                        <!-- Includes its own copy of an old version of the servlet API-->
+                                        <exclude>org.apache.tomcat.embed:tomcat-embed-core</exclude>
+                                        <exclude>org.jboss.modules:jboss-modules</exclude>
+                                        <!-- We prefer ByteBuddy where possible -->
+                                        <exclude>org.javassist:javassist</exclude>
                                     </excludes>
```

Comment:
I have noticed the discussion about moving to `ExternalRules` eventually, but it seems we lost the exclusion for `org.ow2.asm:asm`?

Reply:
`org.ow2.asm:asm` some tests were failing for me when I excluded it for all modules. IIRC, originally it was excluded only for some specific module. Do you know of any specific reason why `org.ow2.asm:asm` should be banned?

Reply:
It was excluded due to #226 (and banned in a later PR).

The runtime part of quarkus-cxf should not depend on asm.